components/badge/badge_pins.h

```diff
@@ -140,6 +140,40 @@
 #define MPR121_PIN_NUM_LEDS     10
 #define MPR121_PIN_NUM_SD_CD    11
 
+#elif defined(CONFIG_SHA_BADGE_V3_LITE)
```

Comment:
Sure, the question is just if it'd be an improvement: removing the duplication will make the code (slightly) less straightforward.

I'm OK with removing the duplication, but it's not necessary IMHO.